atlascope/core/models/investigation.py

```diff
@@ -56,30 +56,22 @@ class InvestigationDetailSerializer(serializers.ModelSerializer):
     owner = serializers.SerializerMethodField('get_owner')
     investigators = serializers.SerializerMethodField('get_investigators')
     observers = serializers.SerializerMethodField('get_observers')
-    datasets = serializers.SerializerMethodField('get_datasets')
-    pins = serializers.SerializerMethodField('get_pins')
 
     def get_owner(self, obj):
         return obj.owner.username or None
 
-    def get_investigators(self, obj):
+    def get_investigators(self, obj) -> list:
```

Comment:
This should specify a more specific return type, `List[...]` (with an appropriate type filled in for that ellipsis).